src/Microsoft.Health.Dicom.Core/Features/Store/StoreDatasetValidator.cs

```diff
@@ -244,13 +247,14 @@ string EnsureRequiredTagIsPresent(DicomTag dicomTag)
             {
                 // validate with additional leniency
                 var validationWarning =
-                    dicomDataset.ValidateDicomTag(requiredCoreTag, _minimumValidator, withLeniency: true);
+                    dicomDataset.ValidateDicomTag(requiredCoreTag, _minimumValidator, validationStyle: ValidationStyle.Default);
```

Comment:
I take it we're leaving the larger "rule" refactor for later?

Reply:
Looking at this line, it feels why can't minivalidator know about the validation style?